emulators/xhyve/Portfile

```diff
@@ -19,9 +19,15 @@ long_description \
     It can run FreeBSD and vanilla Linux distributions and \
     may gain support for other guest operating systems in the future.
 
-checksums           rmd160  b1e803cad9b1577925bee4230efee8a1a198cea0 \
-                    sha256  8bc4198bea593f56676b17a36a2a2b1b6490f7aa26c0e06d70c501b79db3d77d \
-                    size    11718059
+checksums           rmd160  a8de75267e1ff10da9d9957aa68f582867fae65b \
+                    sha256  403b43e4d157ccdd5275fcce034afeb0f8a7804c029d7dcf670f1dcc7929a55f \
+                    size    11718076
+
+patch.pre_args      -p1
+patchfiles          \
+    add-hypervisor-elements.diff \
+    disable-VMX-big-sur.diff
+
```

Comment:
please align the patchfiles in the same was as the rest of the Portfile and remove the extraneous newline